week-8/learning-outcomes.md

```diff
@@ -0,0 +1,28 @@
+# Week 8 Learning Objectives
+
+## Full stack app
+
+### Client rendering
+
+- [ ] We can make `fetch` requests to our own server for JSON data
+- [ ] We can render elements in the DOM based on the server data
+
+### Error handling
+
+- [ ] We can respond with useful error information to the client
+- [ ] We can throw errors when our server hits a problem
+- [ ] We can catch errors and log them correctly
+- [ ] We can handle errors that occur in async code
+- [ ] We can show the user useful messages based on server errors
+
```

Comment:
Is it asking too much that, *"We can write automated tests for our error handling"*?

Reply:
No I think that's a great idea. They should know how to mock by this point, so it wouldn't be unreasonable to ask them to mock an error response from their API and test that they show a relevant message to the user.